script.js

```diff
@@ -107,6 +107,14 @@ function handleClickOption(event) {
 
 function renderQuestion() {
   const { question } = store;
+  var questionHeader = document.querySelector('.quiz-question');
```

Comment:
```suggestion
  var questionHeader = document.querySelector('.js-quiz-question');
```

It's a good practise to only target DOM nodes with js-prefixed classes. That way if classname is changed for style purposes the JS logic won't break. If you remove the js- class you have to manually check if JS logic is working.

Reply:
note I only query selecting DOM nodes. If you want to add a class to that DOM node that class doesn't have to be prefixed with `js-`. So adding class `fade-in` is fine.